src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -81,6 +85,32 @@ public long claimCount(UUID playerUUID) {
         return getClaimsOf(playerUUID).size();
     }
 
+    @StringProvider(
+            text = "Claim Type",
+            description = "Basic + Town + SUBDIVISION + Admin",
+            iconName = "map-marker",
+            iconColor = Color.BLUE,
+            showInPlayerTable = true
+    )
+    public String claimType(UUID playerUUID) {
+        if (getClaimsOf(playerUUID) != null) {
```

Comment:
Avoid duplicate call to `getClaimsOf` by declaring the returned value as a variable - multiple calls can be very expensive if the other plugin uses database to get these claims.